pages/home/oai-agents/overview.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: "Arcade with OpenAI Agents overview"
+description: "Comprehensive guide to using Arcade with the OpenAI Agents library"
+---
+
+# Arcade with OpenAI Agents
+
+The `agents-arcade` package provides a seamless integration between [Arcade](https://arcade.dev) and the [OpenAI Agents Library](https://github.com/openai/openai-python). This integration allows you to enhance your AI agents with powerful Arcade tools including Google Mail, LinkedIn, GitHub, and many more.
+
+## Installation
+
+Install the necessary packages to get started:
+
+```bash
+pip install agents-arcade arcadepy
+```
+
+Make sure you have your Arcade API key ready. [Get an API key](/home/api-keys) if you don't already have one.
+
+## Key features
+
+- **Easy integration** with the OpenAI Agents framework
+- **Access to all Arcade toolkits** including Google, GitHub, LinkedIn, X, and more
+- **Create custom tools** with the Arcade Tool SDK
+- **Manage user authentication** for tools that require it
+- **Asynchronous support** compatible with OpenAI's Agent framework
+
+## Basic usage
+
+Here's a simple example of using Arcade tools with OpenAI Agents:
+
+```python
+from agents import Agent, Runner
+from arcadepy import AsyncArcade
+from agents_arcade import get_arcade_tools
+from agents_arcade.errors import AuthorizationError
+
+async def main():
+    # Initialize the Arcade client
+    client = AsyncArcade()
```

Comment:
when i was trying this out, I hardcoded my api key `client = AsyncArcade(api_key="...")`, but got this error `agents.exceptions.UserError: Error running tool Google_ListEmails: The api_key client option must be set either by passing api_key to the client or by setting the ARCADE_API_KEY environment variable`. Is this a bug?